docs/CHANGELOG.md

```diff
@@ -6,7 +6,22 @@ Socketless _strictly_ adheres to [semver](https://semver.org)'s major.minor.patc
 - minor version changes indicate new functionality that does not break backward compatibility,
 - major version changes indicate backward-incompatible external API changes, no matter how small.
 
-# Current version history
+# Current version
+
+## v4.1.0 (30 November 2024)
+
+Changed the init sequence for browser clients. The code now waits to call `init()` on the web client until a socket connection has been established, _and_ the current state has been obtained from the client and locally bound.
+
+This changes gives code inside the `init()` function access to an up-to-date `this.state` variable, and allows web clients to immediately build the correct UI, rather than needing to first generate a "default" UI that cannot be updated to the correct view until `updateState` happens.
```

Comment:
```suggestion
This gives code inside the `init()` function access to an up-to-date `this.state` variable, and allows web clients to immediately build the correct UI, rather than needing to first generate a "default" UI that cannot be updated to the correct view until `updateState` happens.
```